diff --git a/dist/schemas.js b/dist/schemas.js
index 060d0b98b76515e4f0a1cd9edb3c3f5d31597540..f17054ec71086d81f8242282a5e0bb42adbcb657 100644
--- a/dist/schemas.js
+++ b/dist/schemas.js
@@ -65,7 +65,7 @@ const LayoutPropsSchema = z.strictObject({
   banner: reactNode.optional().meta({
     description: "Rendered [`<Banner>` component](/docs/built-ins/banner). E.g. `<Banner {...bannerProps} />`"
   }),
-  children: reactNode,
+  children: reactNode.optional(),
   copyPageButton: z.boolean().default(true).meta({
     description: "Hide/show copy page content button."
   }),
diff --git a/dist/types.generated.d.mts b/dist/types.generated.d.mts
index daa88cd96a942e16e74cc83d58da58500b9ab858..f381a304a531f9385906a821448018f8a7364f17 100644
--- a/dist/types.generated.d.mts
+++ b/dist/types.generated.d.mts
@@ -5,7 +5,7 @@ interface LayoutProps {
      * Rendered [`<Banner>` component](/docs/built-ins/banner). E.g. `<Banner {...bannerProps} />`
      */
     banner?: React.ReactNode;
-    children: React.ReactNode;
+    children?: React.ReactNode;
     /**
      * Hide/show copy page content button.
      * @default true
